src/styles/_catalog.scss

```diff
@@ -14,6 +14,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    @include smooth-transition;
```

Comment:
specify which property you're animating

Reply:
Not fixed. You should specify what props you want to make smooth, for exmpl `transition: transform 300ms, color 300ms;`